### ISSUE TITLE
fixture 'requests_mock' not found with Pytest

### ISSUE THREAD

**Issue by rexwangcc** [jamielennox/requests-mock#56]
I ran into this error with Pytest:
```
E       fixture 'requests_mock' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, pytestconfig, record_property, record_xml_attribute, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.
```

the versions:
```
pytest==3.5.0
pytest-cov==2.5.1
requests==2.18.4
requests-mock==1.4.0
requests-oauthlib==0.8.0
```

Any insights here?

**Comment by jamielennox** [comment]
Yes, basically i've merged the code in #51 but I haven't done a release yet. It'll be available in 1.5.0. I was thinking I was going to get time to work on another part for this release that I haven't been able to. 

I'll do a release in the next day or two (or maybe just now).

**Comment by jamielennox** [comment]
released, let me know how that goes.
